Automattic/wp-calypso
# ETK Global styles: add new fonts

#### Proposed Changes

Takes font list from [Jetpack Global Styles implementation](https://github.com/Automattic/jetpack/blob/5989c598b3cecc5a30ed8b420ea44ea958a68e5f/projects/plugins/jetpack/modules/google-fonts.php#L22-L69) which I've also just [updated to include new fonts.](https://github.com/Automattic/jetpack/pull/27441).

See p9Jlb4-5Dj-p2 for design discussion.

#### Testing Instructions

* Upload ETK to Simple & Atomic
* Have an older site with classic theme, e.g. "Varia" theme.
* Open page editor, and "global styles" sidebar. Apply some fonts:

    <img width="1156" alt="Screenshot 2022-12-21 at 11 12 18 1" src="https://user-images.githubusercontent.com/87168/208870992-f1bac119-85f7-4525-96df-19749f407bbc.png">

* Confirm that fonts load from WP.com in the editor and at the frontend of the site.

#### Pre-merge Checklist





Related to #
